pkg/shardddl/optimism/keeper.go

```diff
@@ -66,10 +68,21 @@ func (lk *LockKeeper) RebuildLocksAndTables(
 			}
 		}
 	}
+
+	// update missTable's table info for locks
```

Comment:
I think we could remove `missTable`, and pass `initSchemas` here to do same logic. OK to not change because time maybe not enough